test/unittests/execute_floating_point_test.cpp

```diff
@@ -990,7 +990,7 @@ TYPED_TEST(execute_floating_point_types, add_sub_neg_relation)
     constexpr auto sub_fn_idx = 0 + fn_offset;
     constexpr auto addneg_fn_idx = 1 + fn_offset;
 
-    auto instance = instantiate(std::move(module));
+    auto instance = instantiate(module);
```

Comment:
Why is the `std::move` dropped?

Reply:
To keep `unittests` consistently not using move semantics.

Reply:
I don't think this is good consistency.